sql-composer/src/composer/direct.rs

```diff
@@ -128,9 +127,7 @@ mod tests {
             data:         None,
         };
 
-        let (remaining, insert_stmt) = parse_template(Span::new("INSERT INTO person (name, time_created, data) VALUES (:bind(name), :bind(time_created), :bind(data));".into()), None).unwrap();
-
-        assert_eq!(remaining.fragment, "", "nothing remaining");
+        let insert_stmt = SqlComposition::parse("INSERT INTO person (name, time_created, data) VALUES (:bind(name), :bind(time_created), :bind(data));", None).unwrap();
```

Comment:
parse() doesn't return remaining like parse_template did.  Does it check internally that any remaining.fragment is empty?

Reply:
Oh, that reminds me, checking this and ensuring that an ending is found is on the 0.1.0 checklist. It also reminds me I need to start that checklist.